test/integrationtest.py

```diff
@@ -103,7 +103,7 @@ def __init__(self, command):
         time.sleep(2)
 
     def __del__(self):
-        print("Stopping command %s" % self.__open)
+        print(("Stopping command %s" % self.__open))
```

Comment:
What does this do?

Reply:
I don't think I changed something here, but it's just wrong and will print out the string inside a tuple with length one ;-)